catalyst_voices/apps/voices/integration_test/Earthfile

```diff
@@ -3,7 +3,7 @@ VERSION 0.8
 IMPORT ../../.. AS catalyst-voices
 
 integration-test-web:
-    FROM catalyst-voices+build-web
+    FROM catalyst-voices+builder
```

Comment:
`flutter drive` which is used by this target always compiles it's own app, therefore it doesn't make sense to depend on build-web which compiles a production app, it won't be used.